packages/salesforcedx-vscode-core/src/orgBrowser/metadataOutlineProvider.ts

```diff
@@ -137,7 +137,11 @@ export class MetadataOutlineProvider
 }
 
 function parseErrors(error: string): Error {
-  const e = JSON.parse(error);
+  const sanitized = error.substring(
```

Comment:
let's add a test for this scenario.

Reply:
I might suggest putting this JSON sanitization snippet into a util since we're doing this anytime we collect JSON output from a CLI command. See `orgCreateResultParser.ts` and `deployResultParser.ts`

Reply:
Yup, we've logged another story to move this code into a util once Jonny finishes his change (which also uses the sanitization) as well.